src/python-frontend/python_converter.cpp

```diff
@@ -201,6 +202,17 @@ void python_converter::adjust_statement_types(exprt &lhs, exprt &rhs) const
   }
 }
 
+std::string python_converter::create_symbol_id() const
+{
+  std::string symbol_id = "py:" + python_filename;
+  if (!current_class_name.empty())
+    symbol_id += "@C@" + current_class_name;
```

Comment:
The + operator is usually okay for small string operations. However, it might be more efficiently done using std::stringstream or std::string::append to avoid unnecessary copying.

Reply:
I changed to stringstream. Thanks.